src/containers/Storage/StorageGroups/StorageGroups.tsx

```diff
@@ -120,6 +120,24 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes}: StorageGro
             align: DataTable.LEFT,
             defaultOrder: DataTable.DESCENDING,
         },
+        {
+            name: TableColumnsIds.UsedPercents,
+            header: tableColumnsNames[TableColumnsIds.UsedPercents],
+            width: 100,
+            render: ({row}) => {
+                const usage = getUsage(row, 5);
+                return (
+                    <Label
+                        theme={getUsageSeverity(usage)}
+                        className={b('usage-label', {overload: usage >= 100})}
```

Comment:
I suggest to discuss edge value - maybe 95% is already critical?

Reply:
Critical is 85, and is colored red. 100 and greater is just redder :)

Reply:
But I add this to the questions that need discussing regarding this feature, thanks